### DISCUSSION TITLE
How to add a live graph

### DISCUSSION THREAD

**Discussion by boatloader** [TomSchimansky/CustomTkinter#1328] [Q&A]
Hi there,
I'm trying to find a way to implement a GUI with a live updating graph, a button and maybe an indicator.
The graph is updated from a tcp socket with a list of values every 250-500ms.
Has someone an idea, how I can integrate such a graph into the complex GUI example, as given by Tom?
Best regards

**Comment by ElectricCandlelight** [comment]
What sort of graph?  Plus what do you mean by an indicator?

**Reply by boatloader** [reply]
The graph would be a spectrogram (FFT output with 8192 bins) and the indicator means just a "light" that shows, if a specific frequency is present or not :)

**Reply by MaxNiark** [reply]
Hello, 

in canvas, u can place dot, , just write def (running all 250ms) and input the coord of dot in your canvas, 

one by one, without clear, but i don't know if dat can work at 250ms ^^
